src/Infrastructure/LeanCode.Logging/LeanCodeLoggingBuilderExtensions.cs

```diff
@@ -0,0 +1,29 @@
+using LeanCode.Logging;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Extensions.DependencyInjection;
+
+public static class LeanCodeLoggingBuilderExtensions
+{
+    public static ILoggingBuilder ClearLeanCodeLoggers(this ILoggingBuilder builder)
```

Comment:
YAGNI?

Reply:
It could be useful for cases where we are handed a preconfigured host builder and then we want to silence its log output before starting it. Perhaps in integration tests?

But it's nothing that can't be achieved with manual manipulation of the service collection so the value added is indeed minimal… I'll remove it and then consider reintroducing it once I actually find a place where I could use it.